documentation/Data-View.md

```diff
@@ -35,15 +35,16 @@ Returns an array of data table views the user is allowed to see (based on JWT he
     {
         "tableName": "user",
         "title": "Users (Restricted View)",
-        "code": "userRestricted"
+        "code": "userRestricted",
+        "urlSlug":"user-restricted"
     },
     ...etc.
 ]
 ```
 
 This endpoint is called in order to populate the Database menu in the UI.
 
-### `/data-views/table/<tableName>?<queries>`
+### `/data-views/<dataViewCode>?<queries>`
 
 For querying a specific table. Data is returned in the following structure:
```

Comment:
Probably needs updating to match the latest response

Reply:
What response, sorry?

Reply:
The response shape hasn't changed, I don't think.